kong/plugins/datadog/handler.lua

```diff
@@ -89,7 +89,7 @@ end
 
 local DatadogHandler = {
   PRIORITY = 10,
-  VERSION = "3.0.1",
+  VERSION = "3.1.0",
```

Comment:
What is changing besides the version; I don't see any underlying plugin changes here or in the `release/2.5.x` branch from the last `2.5.0` release.

Reply:
(commit removed)